external/lein-repo/src/lein_repo/plugin.clj

```diff
@@ -234,18 +244,18 @@
                :test-paths test-paths)
         (update-in [:dependencies] conj ['lein-repo lein-version])
         middleware
+        merge-test-profile
         (update-in [:source-paths] concat test-paths)
         (dissoc :warn-on-reflection)
         (assoc :root repo-root
                :eval-in :subprocess
-               :jvm-opts ^:replace ["-Xmx1000m" "-XX:+UseConcMarkSweepGC" "-XX:+CMSClassUnloadingEnabled" "-XX:MaxPermSize=512M"]))))
+               ;;:jvm-opts ^:replace ["-Xmx1000m" "-XX:+UseConcMarkSweepGC" "-XX:+CMSClassUnloadingEnabled" "-XX:MaxPermSize=512M"]
+               ))))
 
-(def test-all-project
-  (delay (test-all-project*
-          (read-project-file (first (all-internal-deps))))))
+(def test-all-project (memoize test-all-project*))
 
-(defn ordered-source-and-test-paths [& [no-tests?]]
-  (let [mega-project @test-all-project
+(defn ordered-source-and-test-paths [base & [no-tests?]]
+  (let [mega-project (test-all-project base)
```

Comment:
Nice, I was wondering why the code was written this way (not using the project map from the actual project!) anyway.